src/test/java/CreateAccountTest.java

```diff
@@ -2,6 +2,7 @@
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 import pages.CreateAccountPage;
 import pages.UserPanel;
```

Comment:
Please optimize imports and removed not used imports.
It applies to all tests classes in the project.